Cargo.toml

```diff
@@ -7,34 +7,48 @@ edition = "2021"
 members = [".", "tools/genesis-gen"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[workspace.dependencies]
+iroha_client = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
+iroha_data_model = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
+iroha_crypto = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
+iroha_telemetry = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
+iroha_core = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
+iroha_config = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
+color-eyre = "0.5"
+serde = "1.0"
+serde_json = "1.0"
+clap = { version = "3.1.12", features = ["derive", "env"] }
+rand = { version = "0.8.5" }
+faker_rand = { version = "0.1" }
 
 [dependencies]
 actix = "0.13.0"
 actix-web = { version = "4.0.0", default-features = false, features = ["macros", "compress-brotli", "compress-gzip", "cookies"] }
-awc = { version = "=3.0.0", default-features = false, features = ["compress-brotli", "compress-gzip", "cookies"]  }
+awc = { version = "=3.0.0", default-features = false, features = ["compress-brotli", "compress-gzip", "cookies"] }
 actix-server = "2.1.1"
 tracing = "0.1.32"
 tracing-subscriber = { version = "0.3.10", features = ["env-filter"] }
 tracing-log = "0.1.2"
 tracing-actix-web = "0.5.1"
-clap = { version = "3.1.12", features = ["derive", "env"] }
-iroha_client        = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
-iroha_data_model    = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
-iroha_crypto        = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
-iroha_telemetry     = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
-iroha_core          = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
-iroha_config        = { git = "https://github.com/hyperledger/iroha.git", rev = "e3cd89bd2b6bef0d822900eb71a1854c7b86e2aa" }
-color-eyre = "0.5"
-serde = "1.0"
-serde_json = "1.0"
+iroha_client = { workspace = true }
+iroha_data_model = { workspace = true }
+iroha_crypto = { workspace = true }
+iroha_telemetry = { workspace = true }
+iroha_core = { workspace = true }
+iroha_config = { workspace = true }
 futures = "0.3.21"
 http = "0.2.6"
 chrono = { version = "0.4.19", features = ["serde"] }
 hex = "0.4.3"
 parity-scale-codec = "3.2.1"
 thiserror = "1.0"
-rand = { version = "0.8.5", optional = true }
-faker_rand = { version = "0.1", optional = true }
+bytes = "1.4.0"
```

Comment:
Where does `bytes` come from? I don't see it in the dependencies list before the changes.

Reply:
Bytes is the dependency used by ```iroha_core``` in RC16. So think it makes sense not to include this here and add it in the the upcoming RC16 issue